Makefile

```diff
@@ -236,6 +242,11 @@ controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessar
 $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(CONTROLLER_GEN) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION)
 
+.PHONY: yq
+yq: $(YQ) ## Download controller-gen locally if necessary.
+$(YQ): $(LOCALBIN)
+	test -s $(YQ) || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@$(YQ_VERSION)
```

Comment:
Would it make sense to check existence of a system-wide yq, like opm does? Or it stick to the version?

Reply:
I would prefer to have it totally under our control. There are envs where yq v3 can be used as system-wide binary and it has entirely different commands.